specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -146,50 +165,144 @@ model TextBlocklist {
+  blocklistItemIds: string[];
+}
+
+@added(ContentSafety.Versions.v2023_10_15_Preview)
+@doc("Project is an entity to manage deployment with specified policy.")
+@resource("projects")
+model Project {
+  @doc("Project name.")
+  @pattern("^[a-zA-Z][0-9A-Za-z-]*[0-9a-zA-Z]$")
+  @key("projectName")
+  @visibility("read", "query")
+  @maxLength(128)
+  @minLength(3)
+  projectName: string;
+
+  @doc("Project description.")
+  @maxLength(255)
   description?: string;
 
-  @doc("Block item content.")
+  @doc("Project modality.")
+  @visibility("read", "query", "create")
+  modality: ContentModality;
+
+  @doc("Project display name.")
   @maxLength(128)
-  text: string;
+  @minLength(3)
+  displayName?: string;
+
+  @doc("Project created time.")
+  @visibility("read", "query")
+  createdAt?: utcDateTime;
+
+  @doc("Project last modified time.")
+  @visibility("read", "query")
+  lastModifiedAt?: utcDateTime;
+
+  @doc("Deployed text policy. This value must be null if the modality is not Text.")
+  textPolicy?: TextPolicy;
+
+  @doc("Deployed image policy.This value must be null if the modality is not Image.")
+  imagePolicy?: ImagePolicy;
+}
+
+@added(ContentSafety.Versions.v2023_10_15_Preview)
+@doc("Modality type.")
+enum ContentModality {
+  @doc("Text modality.")
+  Text,
+  @doc("Image modality.")
+  Image,
+}
+
+@added(ContentSafety.Versions.v2023_10_15_Preview)
+@doc("TextCategoryPolicy. Content which severity less than the acceptLessThan will be accepted.")
+model TextCategoryPolicy {
+  @doc("Text category.")
+  category: TextCategory;
+
+  @doc("Severity less than the threshold will be treated as accepted for the category. The value can only be 2, 4, 6 for FourSeverityLevels. If the threshold is undefined, it means all content are accepted.")
+  acceptSeverityThreshold?: int32;
+}
+
+@added(ContentSafety.Versions.v2023_10_15_Preview)
+@doc("ImageCategoryPolicy. Content which severity less than the acceptLessThan will be accepted.")
+model ImageCategoryPolicy {
+  @doc("Image category.")
+  category: ImageCategory;
+  
+  @doc("Severity less than the threshold will be treated as accepted for the category. The value can only be 2, 4, 6 for FourSeverityLevels. If the threshold is undefined, it means all content are accepted.")
+  acceptSeverityThreshold: int32;
 }
 
-@doc("The request of adding blockItems to text blocklist.")
-model AddBlockItemsOptions {
-  @doc("Array of blockItemInfo to add.")
-  blockItems: TextBlockItemInfo[];
+@added(ContentSafety.Versions.v2023_10_15_Preview)
+@doc("Policy for the text.")
+model TextPolicy {
+  @doc("The category policy. The length must be larger than 0.")
+  @minItems(1)
+  categoryPolicies: TextCategoryPolicy[];
+
+  @doc("The policy of blocklist.")
+  blocklistPolicy?: BlocklistPolicy;
+
+  @doc("The type of text analysis output. If not assigned, the default value is \"FourSeverityLevels\".")
+  outputType?: AnalyzeTextOutputType = AnalyzeTextOutputType.FourSeverityLevels;
```

Comment:
Similarly, this should be `outputKind`. Same for any other similar use throughout.

Reply:
This concept is to align with the original output type in 10-01 version, which has been GA just yesterday.